### PR TITLE
Lint civicrm.module and civicrm.install files

### PR DESCRIPTION
**Pull request by seamuslee001**

### CODE REVIEW

**Comment by seamuslee001** [comment]
ping @totten @eileenmcnaughton

**Comment by eileenmcnaughton** [comment]
This looks right to me but there is some changing of functions & moving that seems right, but I'd prefer someone more drupally like @jackrabbithanna or @xurizaemon reviewed

File: civicrm.install [hunk 1]
```diff
@@ -32,10 +32,40 @@
  * $Id$
  *
  */
-function civicrm_install() {}
 
 /**
- * Implementation of hook_uninstall( )
+ * Enable CiviCRM.
+ */
+function civicrm_enable() {
```
Comment (seamuslee001) [inline civicrm.install:39]: Moveing of hook_enable to .install file as per http://www.drupalcontrib.org/api/drupal/drupal%21modules%21system%21system.api.php/function/hook_enable/7 "The hook is called every time the module is enabled. It should be implemented in the module's .install file. The implementation can rely on the .module file being loaded."

**Comment by seamuslee001** [comment]
Jenkins test this please

**Comment by jackrabbithanna** [comment]
This is all very safe to merge

**Comment by jackrabbithanna** [comment]
Jenkins test this please

**Comment by jackrabbithanna** [comment]
Merging #467

**Pull request merged**